scope

```diff
@@ -181,6 +187,7 @@ EOF
         if ! check_docker_for_mac || check_probe_only ; then
             check_not_running $SCOPE_CONTAINER_NAME $SCOPE_IMAGE_NAME
             docker rm -f $SCOPE_CONTAINER_NAME >/dev/null 2>&1 || true
+            create_plugins_dir
```

Comment:
However, it turns out that this had been _undepricated_ more recently (https://github.com/docker/docker/issues/21652#issuecomment-203654182). I think we should still keep it here anyway, at least for consistency with the Docker for Mac code path.